helpers/hdocker.py

```diff
@@ -964,6 +964,104 @@ def run_dockerized_pandoc(
         ret = None
     return ret
 
+# #############################################################################
+# Dockerized extract_notebook_images.
+# #############################################################################
+
+def run_dockerized_notebook_image_extractor(
+    notebook_path: str,
+    output_dir: str,
+    *,
+    force_rebuild: bool = False,
+    use_sudo: bool = False,
+) -> None:
+    """
+    Run NotebookImageExtractor in a Docker container.
+
+    """
+    _LOG.debug(hprint.func_signature_to_str())
+    # Build the container image, if needed.
+    container_image = "tmp.notebook_image_extractor"
+    dockerfile = r"""
+    FROM --platform=linux/amd64 python:3.10-slim
```

Comment:
This might be problematic on MacOS / ARM. Can you add a TODO(gp) to fix it for me?